src/lamp_py/bus_performance_manager/events_tm.py

```diff
@@ -165,7 +209,11 @@ def generate_tm_events(tm_files: List[str]) -> pl.DataFrame:
                 pl.col("TRIP_SERIAL_NUMBER").cast(pl.String).alias("trip_id"),
                 pl.col("GEO_NODE_ABBR").cast(pl.String).alias("stop_id"),
                 pl.col("PATTERN_GEO_NODE_SEQ").cast(pl.Int64).alias("tm_stop_sequence"),
+                pl.col("tm_planned_sequence_end"),
```

Comment:
Can you add some more detail to either the asana ticket or PR on why these tables/columns are being added to the `tm_stop_crossings` output. 

I understand pulling some new tables from the TM db and saving them as parquet files, but not why they would need to be added to any bus_performance_manager output.